app/Http/Middleware/AdminMiddleware.php

```diff
@@ -3,11 +3,16 @@
 namespace App\Http\Middleware;
 
 use Closure;
+use Illuminate\Http\RedirectResponse;
 use Illuminate\Http\Request;
+use Symfony\Component\HttpFoundation\Response;
 
 class AdminMiddleware
 {
-    public function handle(Request $request, Closure $next)
+    /**
+     *  @param  Closure(Request): (Response)  $next
```

Comment:
Are parentheses around `Response` needed?

Reply:
@Plytas Not 100% sure, it is from the 10.x skeleton 

https://github.com/laravel/laravel/blob/32ecad53a9a59d7ad33dbfd6352a5ea0fd0da2cf/app/Http/Middleware/RedirectIfAuthenticated.php#L16C103-L16C103

Reply:
I'm pretty sure it's not needed.

Reply:
It does not complain so we will go with no parentheses.